dist/js/medium-editor.js

```diff
@@ -306,6 +308,28 @@ else if (typeof define === 'function' && define.amd) {
             return this;
         },
 
+        /**
+         * Calls an init() function on all registered extensions
+         * Checks whether init() exists before calling
+         *
+         */
+        initExtensions: function () {
```

Comment:
I removed the `initExtensions()` method and replaced it with the `callExtensions('init')` helper method like you suggested.

I had to add `return this` to the helper method.